src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <p class="summaty-title">
+            ${this.title}
+          </p>
+        </div>
+        <div class="summary-icon ${this.expanded ? "expanded" : ""}">
+          <svg viewBox="0 0 24 24">
+            <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"></path>
+          </svg>
+        </div>
+      </div>
+      <div class="container ${this.expanded ? "expanded" : ""}">
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  private _togglePanel(): void {
+    this.expanded = !this.expanded;
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: block;
+        width: 100%;
+        box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
```

Comment:
Don't need. I think, personally, we should build this as a base element with the functionality only and let whatever is using it place it in a `ha-card` or not.

Reply:
This is how the current version looks inside the automation editor:
![image](https://user-images.githubusercontent.com/1741838/92569307-be42b580-f280-11ea-9b95-cd62b70c0503.png)

after removing the padding from `card-content` and adding it to `paper-dropdown-menu-light` and `card-menu` this is the result:
![image](https://user-images.githubusercontent.com/1741838/92569522-08c43200-f281-11ea-8a20-8c5783d0481c.png)

My idea is to show the Time part is at least one of two (`after` and `before`) is filled. same with the Days part - if at least one day is selected then show that part.

Not sure if I should alter `ha-card` styles. Any suggestions?

Reply:
Dont use Ha-card

Reply:
I placed my panel inside ha-card, I'm not using ha-card in my component.

Reply:
You should do that. Cards inside of cards are bad

Reply:
I dont think we should add the box shadow right?

Reply:
done